### PR TITLE
Ankit | Stage | Common Stage PR

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cwm3cyb
https://app.clickup.com/t/86cwhrtjh
https://app.clickup.com/t/86cvbjqjp
https://app.clickup.com/t/86cwgvbru
https://app.clickup.com/t/86cwmjtqy
https://app.clickup.com/t/86cw9ukk8
https://app.clickup.com/t/86cwnebf5
https://app.clickup.com/t/862k8c9x8
https://app.clickup.com/t/86cwg7mr1
https://app.clickup.com/t/86cwcvcnz
https://app.clickup.com/t/86790cgwk
https://app.clickup.com/t/85zu3g48t
https://app.clickup.com/t/86cw55udr
https://app.clickup.com/t/86cvwurc9
https://app.clickup.com/t/86cwmcn5k
https://app.clickup.com/t/86cwpg6yk
https://app.clickup.com/t/86cwhrdah
https://app.clickup.com/t/86cwfw4ea
https://app.clickup.com/t/86cw4fmrm

### CODE REVIEW

**Review by dvCodeWorld — DISMISSED** [review]

File: apps/web-giddh/src/app/vat-report/obligations/obligations.component.scss [hunk 1]
```diff
@@ -16,4 +16,11 @@
     .action-btn {
         min-width: 125px;
     }
+    min-height: calc(100vh - 90px);
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/vat-report/obligations/obligations.component.scss]: move in line no 2

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vat-report/obligations/obligations.component.scss (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/vat-report/obligations/obligations.component.scss]: done

**Review by dvCodeWorld — DISMISSED** [review]

**Review by dvCodeWorld — DISMISSED** [review]

**Review by dvCodeWorld — DISMISSED** [review]

**Review by dilpreetsinghofficial — DISMISSED** [review]

**Review by dilpreetsinghofficial — DISMISSED** [review]

File: apps/web-giddh/src/app/add-company/add-company.component.ts [hunk 2]
```diff
@@ -1251,6 +1251,7 @@ export class AddCompanyComponent implements OnInit, AfterViewInit, OnDestroy {
             delete this.company.creatorSuperAdmin;
         }
         this.company.otherBusinessNature = this.secondStepForm.value.businessNature === "Other" ? this.secondStepForm.value.otherBusinessNature : "NA";
+        this.nextStepForm();
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/add-company/add-company.component.ts:1254]: move into html

File: apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html [hunk 3]
```diff
@@ -844,15 +840,10 @@ <h3>{{ activeCompany?.baseCurrencySymbol }} {{ overallTotal | giddhCurrency }}</
                                                                     ?.value
                                                             ]
                                                         "
-                                                        [defaultValue]="
-                                                            senderWarehouses[
-                                                                branchTransferCreateEditForm.get('sources.0.uniqueName')
-                                                                    ?.value
-                                                            ]?.length === 1
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html:851]: check this

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/models/db/db.ts [hunk 4]
```diff
@@ -231,10 +231,8 @@ class AppDatabase extends Dexie {
             res.aidata[entity] = this.getSlicedResult(entity, arr, isCompany);
             // do entry in db and return all data
             return this.companies.put(res).then(() => {
-                setTimeout(() => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/models/db/db.ts:234]: why removed this?

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 5]
```diff
@@ -601,7 +601,7 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
         });
 
         // region subscribe to last state for showing title of page this.selectedPage
-        this.store.pipe(select(s => s.session.lastState), take(1)).subscribe(s => {
+        this.store.pipe(select(s => s.session.lastState), takeUntil(this.destroyed$)).subscribe(s => {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:604]: what's the need of changing this logic?

File: apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html [hunk 6]
```diff
@@ -479,11 +479,7 @@ <h2>
                                                             )?.value
                                                         "
                                                         [defaultValue]="
-                                                            destinationWarehouses[
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html:482]: why changing this logic? and how is it useful?

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/models/db/db.ts (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/models/db/db.ts:234]: No need of set timeout here.

File: apps/web-giddh/src/app/shared/header/header.component.ts (hunk 5 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/shared/header/header.component.ts:604]: this is required

File: apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/new-inventory/component/branch-transfer/create-branch-transfer/create-branch-transfer.component.html:851]: done

**Review by ravinderwalkover — APPROVED** [review]

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**